enginetest/evaluation.go

```diff
@@ -621,23 +621,24 @@ func WidenRow(sch sql.Schema, row sql.Row) sql.Row {
 	return widened
 }
 
-func widenJSONValues(val interface{}) types.JSONValue {
+func widenJSONValues(val interface{}) sql.JSONWrapper {
 	if val == nil {
 		return nil
 	}
 
-	js, ok := val.(types.JSONValue)
+	js, ok := val.(sql.JSONWrapper)
 	if !ok {
 		panic(fmt.Sprintf("%v is not json", val))
 	}
 
-	doc, err := js.Unmarshall(sql.NewEmptyContext())
-	if err != nil {
-		panic(err)
+	doc := js.ToInterface()
+
+	if _, ok := js.(sql.Statistic); ok {
+		delete(doc.(map[string]interface{})["statistic"].(map[string]interface{}), "created_at")
 	}
```

Comment:
(minor) this is probably worth a comment explaining why we do this here since it's so specific to the `sql.Statistic` type. I'm assuming it's just so we don't have dates and can compare the json structures more easily, but probably still worth a comment since it seems a little out of place with the rest of the code.